### DISCUSSION TITLE
Update zotonic docker compose to include certs volume

### DISCUSSION THREAD

**Discussion by geoidesic** [zotonic/zotonic#3950] [Ideas]
I would recommend this addition to the docker-compose.yml file. It will assist those who with to add the certs to their keychain etc. without having to go hunting.
<img width="735" alt="Screenshot 2024-11-24 at 11 19 04" src="https://github.com/user-attachments/assets/ec790ed8-ddb8-4ab7-9b44-520ccf8a3bba">

**Comment by mworrell** [comment]
Just to be sure: This would place all certificates from the Zotonic docker container in the directory "certs" of the host?

**Reply by geoidesic** [reply]
Yes. I think that's safe for development. Do you see any problem with it?

**Reply by geoidesic** [reply]
The Zotonic documentation (https://zotonic.com/docs/1411/docker) says this: `Zotonic is running with a self-signed certificate. The certificate can be found in docker-data/security/self-signed/.` – but that security folder is empty. 

Maybe mapping the volume to that instead of `certs` makes more sense. It seems that might have been the original intent but for whatever reason that security folder is empty in the current version. How were you mapping it previously? Or has it always been accidentally empty?

Either way, the dev will need the mapping to work in order to trust the certificates on their device.

If you're happy for me to add the following volume mapping, let me know and I'll raise a PR:
```yml
   - ./docker-data/security:/home/zotonic/.config/zotonic/security
```
